src/solve.jl

```diff
@@ -100,62 +113,78 @@ function _solve(cp::ControlledPlant,
         "the dimension of the initial states of the initial-value problem to " *
         "be $(n + m + q), but it is $(dim(Q₀))"))
 
-    if m > 0
-        W₀ = project(Q₀, in_vars)
-    end
+    W₀ = m > 0 ? project(Q₀, in_vars) : nothing
 
-    ti = tstart(time_span)
-    Δti = zero(ti)
-    NSAMPLES = ceil(Int, diam(time_span) / sampling_time)
+    function nnet_forward(X)
```

Comment:
i would outsource this function outside `_solve`

Reply:
The reason to have it inside was to not having to pass `preprocessing` and `normalization` around.

Reply:
I outsourced the function.